app/components/Transactions/Transaction/index.js

```diff
@@ -44,19 +46,20 @@ class Transaction extends Component {
       'transaction__number--pending': tx.pending,
       'transaction__number--positive':
         (tx.type === RECEIVE
-        || tx.type === COINBASE
-        || tx.type === REVEAL
-        || tx.type === REDEEM
-        || tx.type === REGISTER)
-         && !tx.pending,
+          || tx.type === COINBASE
+          || tx.type === REVEAL
+          || tx.type === REDEEM
+          || tx.type === REGISTER)
+        && !tx.pending,
       'transaction__number--neutral':
         (tx.type === UPDATE
-        || tx.type === RENEW
-        || tx.type === OPEN)
+          || tx.type === RENEW
+          || tx.type === OPEN
+          || tx.type === TRANSFER)
```

Comment:
I think FINALIZE would go here, same as below we need a FINALIZE clause after line 117